ipatests/test_integration/test_replica_promotion.py

```diff
@@ -47,6 +48,32 @@ def test_kra_install_master(self):
         assert(found > 0), result2.stdout_text
 
 
+def test_sssd_config_allows_ipaapi_access_to_ifp(host):
```

Comment:
Can you drop the test_ prefix? It makes it look like this is a test rather than a standalone function.

Reply:
I believe this is addressed in commit adc5e44.